full-node/sov-ledger-rpc/src/lib.rs

```diff
@@ -0,0 +1,79 @@
+#![forbid(unsafe_code)]
```

Comment:
let's add `#![deny(missing_docs)]`

Reply:
Yeah, initially I didn't want to document all the `Rpc` trait methods because it would just be a duplication of the `LedgerRpcProvider` documentation, but it makes sense. Will address

Reply:
Fixed in https://github.com/Sovereign-Labs/sovereign-sdk/pull/1032/commits/fd55cbdd85cabc950db3a3cc20226af0705ced33.